.github/scripts/install-slices/install_slices.py

```diff
@@ -359,43 +397,41 @@ def main() -> None:
                 logging.info("  - %s", pkg.package)
     #
     if len(packages) > 0:
-        logging.info("Slices of the following packages will be INSTALLED:")
+        logging.info(
+            "Slices of the following packages will be INSTALLED (parallelization=%s):",
+            cli_args.workers,
+        )
         for pkg in packages:
             logging.info("  - %s", pkg.package)
     else:
         logging.info("No slices will be installed.")
         return
-    # Install the slices in each package.
-    for pkg in packages:
-        # Keep track of whether the copyright file is installed on every "cut"
-        # This should always be the case, whether enforced by a global "essential"
-        # or Chisel itself. Exception: the copyright file will not be installed
-        # if it doesn't exist in the deb itself.
-        missing_copyright = set()
-        for slice in pkg.slices:
-            if cli_args.dry_run:
-                logging.info(
-                    "Installing %s on %s... (--dry-run)",
-                    full_slice_name(pkg.package, slice),
-                    cli_args.arch,
-                )
-            else:
-                install_slice(
-                    pkg.package,
-                    slice,
-                    cli_args.arch,
-                    cli_args.release,
-                    missing_copyright,
-                )
 
-        if len(missing_copyright) > 0:
-            # Does the copyright file exist in the deb?
-            if deb_has_copyright_file(pkg.package):
-                err = "{} has a copyright file but it wasn't installed with: {}".format(
-                    pkg.package,
-                    ",".join(missing_copyright),
-                )
-                raise MissingCopyright(err)
+    # TODO: this list of slices can still be reduced, since many of these
+    # will come as essentials of other slices. We could simply crawl all essentials
+    # in the release, and remove them from the "all_slices" list (since they
+    # are going to get installed anyway). The problem here is accounting: I'd like
+    # to ensure that all slices are still installed nonetheless, even if indirectly,
+    # but that means `install_slices` will have to be aware of the nested essentials.
+    all_slices = [(pkg.package, slice) for pkg in packages for slice in pkg.slices]
+
+    chunk_size = math.ceil(len(all_slices) / cli_args.workers)
```

Comment:
Would there be any point in making sure the number of workers is not > number of slices?